### DISCUSSION TITLE
Fixing rounded corners?

### DISCUSSION THREAD

**Discussion by oxfighterjet** [jgraph/drawio-desktop#1868] [Ideas]
I enjoy using drawio and it works well for my use cases.

I have come across a small implementation detail which I assume was intended to be cleverly designed, but in my opinion works against the user.

When I use rounded corners for the rectangle shape, the radius of the rounded corners changes with the shortest dimension of the rectangle. This is in my opinion not optimal: when I make diagrams, I want my shapes to all share the same radius for the rounded corners, regardless of the size I give my shapes. This is difficult to do when adjusting the rectangle size changes the radius. Additionally, and I may get some minor gripe for bringing this up, I suspect many people coming from tools such as powerpoint also see this as being a counter-intuitive feature, as powerpoint also does not resize the corner radius.

Am I alone in thinking that the rounded corners are probably left unchanged when resizing a given shape?

Thank you for your time!

**Comment by gezamolnar** [comment]
Hi,

   Select the item, then at the "Style" tab (right side) open the "Property - Value" table (below the style settings) and check the "Abs.arc size" (below "Arc Size").

(a bit late to the party :-) )

**Comment by davidb91** [comment]
@gezamolnar , @oxfighterjet  : Thank you very much for the question and the answer, it helps me a lot today because i had the same problem 👍 . Have a good day.
